instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-instrumentation-vertexai-v2"
```

Comment:
I believe this package will have feature parity with https://github.com/traceloop/openllmetry/blob/main/packages/opentelemetry-instrumentation-vertexai/

I think we should publish it as `opentelemetry-instrumentation-vertexai` 

@nirga what would be your thoughts on this?

In particular, OTel can start publishing to v1 (or v2) train and OpenLLMetry can still continue v0 train if necessary.